src/components/CourseCard/CourseCard.tsx

```diff
@@ -38,23 +41,27 @@ const CourseCard = ({ course }: Props) => {
           {course.name}
         </Typography>
         <Typography>{courseDate}</Typography>
-        <Typography>Signups: 0 / {course.maxStudents}</Typography>
+        <Typography>
+          {dict.CourseCard.signups.replace('{x}', String(course.maxStudents))}
```

Comment:
This requires some better way, as it's common to have variables in string

Java has String.format(string, value1, value2) -> perhaps there is equivalent in JavaScript